compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1774,11 +1774,13 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
       {
       case TR::vadd:
       case TR::vsub:
-         if (dt == TR::Int16 || dt == TR::Int8 || dt == TR::Int32 || dt == TR::Float || dt == TR::Double)
+         if (dt == TR::Int8 || dt == TR::Int64)
```

Comment:
It doesn't look like this change was intentional. It's fixed in the next commit, but I think it would be better not to introduce this change at all